src/redux/categories/Categories.js

```diff
@@ -0,0 +1,7 @@
+const Categories = () => (
+  <div className="categoriesContainer">
+    <h1>Under Construction</h1>
+  </div>
+);
+
+export default Categories;
```

Comment:
- [ ] Kindly add action and reducer for categories by using duck pattern, you can make an action `CHECK_STATUS` and in case of action.type is `CHECK_STATUS` the function returns a string "Under construction". As stated in the below requirements.
![image](https://user-images.githubusercontent.com/53402600/184785117-d3bcf057-89e4-4ed8-b968-73f2159c33b4.png)